components/eam/src/physics/crm/crm_physics.F90

```diff
@@ -997,11 +1004,9 @@ subroutine crm_physics_tend(ztodt, state, tend, ptend, pbuf2d, cam_in, cam_out,
             crm_input%ul(icrm,1:pver)     = state(c)%u(i,1:pver)
             crm_input%vl(icrm,1:pver)     = state(c)%v(i,1:pver)
             crm_input%ocnfrac(icrm)       = cam_in(c)%ocnfrac(i)
-#if defined( MMF_ESMT )
-            ! Set the input wind for ESMT
+            ! input wind for ESMT
             crm_input%ul_esmt(icrm,1:pver) = state(c)%u(i,1:pver)
```

Comment:
We don't need to rotate the wind used for ESMT because the ESMT scalars are not the same as the resolved wind on the CRM grid, so it doesn't matter which direction the ESMT vector points.